hadoop-master1/files/etc/trino/conf/node.properties

```
@@ -0,0 +1,3 @@
+node.environment=production
+node.id=trino-coordinator
```

let's use the hostname
```suggestion
node.id=hadoop-master1
```